modelling/src/main/java/org/axonframework/modelling/command/AggregateAnnotationCommandHandler.java

```diff
@@ -323,12 +334,25 @@ public Builder<T> aggregateModel(AggregateModel<T> aggregateModel) {
             return this;
         }
 
+        /**
+         * Sets the {@link CreationPolicyAggregateFactory<T>} for generic type {@code T}. The aggregate factory must produce a
+         * new instance of the Aggregate root based on the supplied Identifier.
+         *
+         * @param creationPolicyAggregateFactory that returns the aggregate instance based on the identifier
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder<T> aggregateFactory(CreationPolicyAggregateFactory<T> creationPolicyAggregateFactory) {
```

Comment:
Nit: let's rename this method to `creationPolicyAggregateFactory` to be in line with the exact component it configures.